index.html

```diff
@@ -29,6 +29,327 @@
     <title>Besufekad's Personal Portfolio</title>
   </head>
   <body>
+    <header class="container_nav">
+      <nav class="header">
+        
+        <div class="mobile_menu">
+          <div class="ham_menu" id="">
+            <a href="#">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </a>
+          </div>
+          <div id="links">
+            <ul>
+             <li><a class="nv" href="#Hello">Hello</a></li>
+            <li><a class="nv" href="#Portfolio">Portfolio</a></li>
+            <li><a class="nv" href="#About_sec">About</a></li>
+            <li><a class="nv" href="#Contact">Contact</a></li>
+            </ul>
+          </div>
+        </div>
+
+
```

Comment:
It would be nice to indent your code probably and remove too many line spaces. Do this in this file.